lib/Doctrine/ODM/MongoDB/Mapping/Driver/AnnotationDriver.php

```diff
@@ -72,12 +70,11 @@ public function loadMetadataForClass($className, \Doctrine\Persistence\Mapping\C
                 $this->addIndex($class, $annot);
             }
             if ($annot instanceof ODM\Indexes) {
-                @trigger_error(
-                    sprintf(
-                        'Indexes annotation used in %s was deprecated in doctrine/mongodb-odm 2.2 and will be removed in 3.0. Specify all Index and UniqueIndex annotations on a class level.',
-                        $className
-                    ),
-                    E_USER_DEPRECATED
+                trigger_deprecation(
+                    'doctrine/mongodb-odm',
+                    '2.1',
```

Comment:
"was deprecated in doctrine/mongodb-odm 2.2"
```suggestion
                    '2.2',
```